src/app/home/shared/forms/transaction-form/transaction-form.component.ts

```diff
@@ -38,15 +43,25 @@ export class TransactionFormComponent implements OnInit {
     this.panelService.sendNewTransactionFormStatus(false);
   }
 
-  saveTransaction(form: NgForm) {
-    console.log('TODO', form);
-  }
-
   getStandardCategoryIcon(budget) {
     return categories.filter(category => category.name === budget.category).map(category => category.icon).join();
   }
 
   checkIfStandardCategory(budget) {
     return categories.map(category => category.name).includes(budget.category);
   }
+
+  selectBudget(budget) {
+    this.selectedBudgetId = budget.id;
+  }
+
+  saveTransaction(form: NgForm) {
+    if (form.form.status === 'VALID') {
```

Comment:
Inverse the conditional statement to return the function before any other code. It helps with not creating unnecessary indent.

Reply:
Ok